index.ios.js

```diff
@@ -23,14 +23,23 @@ const SleekLoadingIndicator = React.createClass({
     loading: React.PropTypes.bool,
   },
 
+  renderText(){
+    if(this.props.text){
```

Comment:
This will always return `true` due to default prop value defined in `getDefaultProps` function.